INSTALL

```diff
@@ -336,8 +336,12 @@
 
   enable-ec_nistp_64_gcc_128
                    Enable support for optimised implementations of some commonly
-                   used NIST elliptic curves. This is only supported on some
-                   platforms.
+                   used NIST elliptic curves.
+                   This is only supported with compilers that:
+                   - support the non-standard type __uint128_t
+                   - define the built-in macro __SIZEOF_INT128__
+                   - support little-endian integers
+                   - tolerate misaligned memory references
```

Comment:
Last two are not compiler's domain. For example "compiler tolerate misaligned memory reference" rather suggests that compiler supports something similar __attribute__((aligned(1)) **and** that code in question actually uses it. Because given the attribute compiler actually can load unaligned say integer even on processor that doesn't tolerate misaligned loads/stores. Simply by loading byte by byte and collecting value with shifts and or-s. Code in question doesn't do that, but assumes that misaligned non-byte loads just work. As for the other, what's "little-endian *integer*"? Endianness is about how non-byte values are stored in memory, not about type or even actual value. In sense that in most cases you simply store and retrieve, and it works on either system, without compiler making doing anything different on either platform.